src/pages/Search/SearchPageBottomTab.tsx

```diff
@@ -64,13 +65,18 @@ function SearchPageBottomTab() {
                     breadcrumbLabel={translate('common.search')}
                     shouldDisplaySearch={false}
                 />
-                <SearchFilters query={query} />
+                <SearchFilters
+                    query={query}
+                    isMobileSelectionModeActive={isMobileSelectionModeActive}
+                />
```

Comment:
Should we move the `isMobileSelectionModeActive` up and make this something like: `!isMobileSelectionModeActive && <SearchFilters`?

Reply:
Great catch! Applied